tools/psu/bd9e302efj.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# Copyright 2021 The Racklet Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Tool for computing component constants for the ROHM Semiconductor BD9E302EFJ.
+# Component naming and computed values apply to the Application Example 2 (Fast load response)
+# in https://fscdn.rohm.com/en/products/databook/datasheet/ic/power/switching_regulator/bd9e302efj-e.pdf.
+
+import itertools
+import iec60063
+import math
+from tabulate import tabulate
+
+
+def avg(*args):
+    return sum(args) / len(args)
+
+
+# BD9E constants
+F_osc = 550000  # Hertz (this can't be changed)
+
+# Variables
+# TODO: Take these as command line arguments?
+V_in = 24  # Volts, input voltage
```

Comment:
V_in_max or V_in? We should probably somehow make it explicit that we're treating this as a range.
Maybe we could also output the min-max values of stuff, e.g. voltage ripple, such that the output has columns min, typical, max instead of just one value